code/game/gamemodes/cult/items/tome.dm

```diff
@@ -90,9 +90,9 @@
 		if(use_check_and_message(scribe))
 			return
 
-		scribe.visible_message(SPAN_CULT("[scribe] slices open their palm with a ceremonial knife, drawing arcane symbols with their blood..."))
-		playsound(scribe, 'sound/weapons/bladeslice.ogg', 50, FALSE)
-		scribe.drip(4)
+		to_chat(scribe, SPAN_CULT("You feel the energy of His presence wash over you as you close your eyes and worshipfully pierce your finger on the spike in the tome's ink vial. It draws and empowers your blood with dark energy, and when it finishes you wipe off your finger on the inside of the tome."))
+		scribe.visible_message(SPAN_WARNING("[scribe] closes their eyes and fiddles with their book for a bit, then takes out a quill and ink vial, using them to draw strange symbols in a red ink..."))
```

Comment:
Visible messages have a `self_message` variable, use that instead of a to_chat

Reply:
Got it!  Will do.

Reply:
Fixed.  Did I do it right?

Reply:
I'll mark this as resolved?  I'm not sure how this works.